addons/jobs/jobs-service/src/main/java/org/kie/kogito/jobs/service/executor/HttpJobExecutor.java

```diff
@@ -39,34 +47,71 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(HttpJobExecutor.class);
 
     @Inject
-    Vertx vertx;
+    private Vertx vertx;
```

Comment:
Use package protected as otherwise quarkus reports warnings

Reply:
Great point, I was not aware of this behavior. I changed the all attributes with @Inject until all warnings were gone.